extensions/pvaPublish/src/node/logger.ts

```diff
@@ -0,0 +1,3 @@
+import logger from 'debug';
+
+export const log = logger('composer:extension:pva-publish');
```

Comment:
The log is already a part of the extension registration. I should have made that clearer, sorry!

Reply:
no worries, I actually understood but I started plumbing it through to all my code and it seemed weird to wrap everything in an outer call and pass the logger around, but I'll fix that